src/server/services/move_servo.py

```diff
@@ -1,5 +1,33 @@
 import logging
+import multiprocessing
+import time
 
 
 def move_servo(angle, duration):
+    """Move servo and return it back without blocking main process
+
+    Parameters
+    ----------
+    angle: int
+        Angle servo will move to.
+    duration: int
+        Duration (in ms) servo will stay turned for.
+    """
     logging.info('Set servo to %i degrees for %ims', angle, duration)
+
+    move = multiprocessing.Process(
+        target=_move_servo, args=(angle, duration))
+    move.start()
```

Comment:
what will happen in this situation:
- move request with duration=1s comes up
- 0.5s delay
- another move request with duration=1s comes up

Will motors stop spinning after 0.5s or after 1s?